### ISSUE TITLE
Option to map all the teams within an organization

### ISSUE THREAD

**Issue by shnitznmore** [alexchantastic/figma-export#4]
Hey
Im working on exporting all the files in my organization, we have multiple teams, each team has multiple teams and each project has multiple files. I cant find a way to map all the files in my organization in a nice json file.

**Comment by alexchantastic** [comment]
Unfortunately, Figma doesn't provide an API to fetch all teams within an organization. It's been a sore spot for plugin developers for a while now. 😢 

I wouldn't mind adding some functionality to the `get-team-files` command to accept multiple team IDs as parameters so you could do something like:

```sh
npm run get-team-files 123 456 789
```

You'd still need to grab each time ID manually though. I'll try to find some time this week to add that in.

In the meantime, you *could* run `get-team-files` manually and save the JSON output and then use [jq](https://jqlang.github.io/jq/) to merge each JSON file together into a single file that the CLI can leverage. I believe you can use this jq command to do so:

```sh
# assuming you are saving the JSON files as files-1.json, files-2.json, etc.
jq -s add files-*.json > files.json
```

**Comment by shnitznmore** [comment]
Yeah I guess having an option to manually write all the team IDs would work. thanks

**Comment by shnitznmore** [comment]
also, would it be possible to save the files like this
Team>Project>file
right now its making a new folder for each project but they are all in the same folder, I would like to organize it like its in Figna where each team has its projects.

**Comment by alexchantastic** [comment]
Yea I think I can refactor it like that if you're coming at the angle of multiple teams in an organization 👍

**Comment by alexchantastic** [comment]
I just published [v2.0.0](https://github.com/alexchantastic/figma-export/releases/tag/v2.0.0) which adds features to support multiple teams.

Unfortunately, Figma doesn't have an API endpoint to grab team names so the team folder will just be named with team's ID.

Let me know if you run into any issues with the new release. Thanks for the suggestions!
